salt/client/mixins.py

```diff
@@ -11,9 +11,11 @@
 import weakref
 
 import salt.exceptions
+import salt.minion
 import salt.utils
 import salt.utils.event
 import salt.utils.jid
+import salt.utils.job
```

Comment:
I don't really like having this separate job lib, since that function is really only used by MasterMinions-- although I'm not sure MasterMinion is the right place to put it...